skbio/metadata/_intersection.pyx

```diff
@@ -0,0 +1,513 @@
+"""
```

Comment:
I added an update method in the bx-python code.  Note that those changes didn't end up getting used - so they could probably be removed from this PR.